# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wr/wrench/reftests/blend/reftest.list
# Commit: 5589923a52c8
# Full Hash: 5589923a52c80ed09c8ce383afb0385cabc0ec18
# Author: Glenn Watson <git@intuitionlibrary.com>
# Date: 2020-12-15 09:29:54
# Regressor Bug: 1682365
# File Overlap Count: 1
# Description:
#   Bug 1682365 - Pt 2 - Remove texture array usage from render task graph. r=nical
#   
#   With this change, all color/alpha intermediate surfaces are individual
#   textures, rather than texture arrays.
#   
# ==============================================================================

diff -r 2d192d43277b -r 5589923a52c8 gfx/wr/wrench/reftests/blend/reftest.list
--- a/gfx/wr/wrench/reftests/blend/reftest.list	Tue Dec 15 00:01:19 2020 +0000
+++ b/gfx/wr/wrench/reftests/blend/reftest.list	Tue Dec 15 00:43:28 2020 +0000
@@ -21,4 +21,4 @@
 fuzzy(1,2502) == transparent-composite-1.yaml transparent-composite-1-ref.yaml
 fuzzy(1,2502) == transparent-composite-2.yaml transparent-composite-2-ref.yaml
 
-fuzzy(2,324) == multi-mix-blend-mode.yaml multi-mix-blend-mode-ref.yaml
+fuzzy(2,411) == multi-mix-blend-mode.yaml multi-mix-blend-mode-ref.yaml